x-pack/plugins/triggers_actions_ui/public/application/components/builtin_alert_types/threshold/expression.scss

```diff
@@ -1,3 +1,11 @@
 .actAlertVisualization__chart {
   height: $euiSize * 15;
 }
+
+.euiComboBoxOptionsList,
+.euiFilterSelectItem,
+.ReactVirtualized__List,
+.ReactVirtualized__Grid__innerScrollContainer {
```

Comment:
@YulNaumenko This is actually quite dangerous as there is no automatic scoping of CSS selectors in Kibana. So these styles will be applied to **all** EuiComboBoxes, EuiFilterSelectItems and so on throughout Kibana.

Instead of overriding the EUI styles, I'd consider the layout of the components and their containers. It may just be better to increase the width of the EuiPopover that contains these combo boxes.

Reply:
Awesome! @cchaos thank you for the idea!